src/js/components/LoginForm.js

```diff
@@ -90,15 +90,20 @@ export default class LoginForm extends Component {
     const center = ! align || 'stretch' === align || 'center' === align;
 
     const errorsNode = errors.map((error, index) => {
-      let errorComponent;
       if (error) {
-        errorComponent = (
+        let errorMessage;
+        if (React.isValidElement(error)) {
+          errorMessage = error;
+        } else {
+          errorMessage = <FormattedMessage id={error} defaultMessage={error} />;
```

Comment:
Thanks for keeping the FormattedMessage for backwards compatibility sake. But in the future we will be removing it. I don't think we should be translating anything here. The caller should make sure to translate this before sending it to us.

This will eventually remove the need for `React.isValidElement` call.